### PR TITLE
feature/radar_svr_plots

### PR DESCRIPTION
**Pull request by LoganDawson-NOAA**
## Pull Request Testing ##

- [ ] Describe testing already performed for this Pull Request:</br>
Jobs have been tested interactively and on cron to ensure proper completion.

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

1. Clone the radar_svr_plots feature branch
2. Clone the repository with cron scripts for testing: https://github.com/LoganDawson-NOAA/EVS_crons. The two drivers are `drive_jevs_cam_radar_plots.sh` and `drive_jevs_cam_severe_plots.sh`.
3. In dev/drivers scripts, update COMIN to point to /lfs/h2/emc/vpppg/noscrub/logan.dawson/evs/v1.0 (or symlink to your user space)

4. Jobs can be run interactively at any time or can be run on cron as follows (need to ensure there's a cron.out directory in the user's ptmp space):
```
# Crons for EVS jobs to generate daily RADAR plots
15 02 * * * /lfs/h2/emc/vpppg/save/${USER}/EVS_crons/drive_jevs_cam_radar_plots.sh 1>/lfs/h2/emc/ptmp/${USER}/cron.out/cam_radar_plots.out 2>&1

# Crons for EVS jobs to generate daily SEVERE plots
15 09 * * * /lfs/h2/emc/vpppg/save/${USER}/EVS_crons/drive_jevs_cam_severe_plots.sh 1>/lfs/h2/emc/ptmp/${USER}/cron.out/cam_severe_plots.out 2>&1
```

A total of 8 tar files are expected in the output.

- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes]**

- [ ] Do these updates/additions include sufficient testing updates? **[Yes]**

- [ ] Please complete this pull request review by **[8/31/23]**.</br>

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

**Comment by LoganDawson-NOAA** [comment]
The associated feature branch has been brought up to date with the develop branch and is ready for testing and review

**Comment by ShelleyMelchior-NOAA** [comment]
Hi @LoganDawson-NOAA  -- I have cloned your repositories and will begin working on testing branch `radar_svr_plots`.

**Comment by ShelleyMelchior-NOAA** [comment]
I have run both radar and severe plots processing. Please review the output. I see 4 tar files but not 8 ... ?
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/plots/cam`
logfiles: `/lfs/h2/emc/ptmp/shelley.melchior/cron.out`

**Comment by LoganDawson-NOAA** [comment]
@ShelleyMelchior-NOAA ohh good catch. I forgot to remove an ex-script setting that prevented the last90days plots from being generated. I just made and validated that fix.

There are attendant changes to the cron drivers in https://github.com/LoganDawson-NOAA/EVS_crons that you'll need to have for all 8 tarballs to be produced

File: dev/drivers/scripts/cam/plots/jevs_cam_radar_plots.sh [hunk 1]
```diff
@@ -1 +1,79 @@
-sample
+#!/bin/bash
+#PBS -N jevs_cam_radar_plots_00
+#PBS -j oe
+#PBS -S /bin/bash
```
Comment (ShelleyMelchior-NOAA) [inline dev/drivers/scripts/cam/plots/jevs_cam_radar_plots.sh:3]: Just a nitpick ... b/c you have the job card `#PBS -S /bin/bash` you do not need to state `#!/bin/bash` at line 1 of the driver scripts.

Comment (LoganDawson-NOAA) [reply dev/drivers/scripts/cam/plots/jevs_cam_radar_plots.sh:3]: just removed that line from both driver scripts included here. I'll remove that line from the driver scripts included in PR #161. I'll include the update on that PR.

**Comment by ShelleyMelchior-NOAA** [comment]
I pulled in the updates in both repositories and re-ran. This time I see 7 new tar files ... ? Still not 8. Maybe missing radar nbrctc last 90 days? It is tricky to see which log output file is for which processing since they all share the same output logfile name.
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/plots/cam/atmos.20230724`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/plots/cam/atmos.20230729`
logfiles: `/lfs/h2/emc/ptmp/shelley.melchior/cron.out/jevs_cam_*` dated Jul 31

evs.plots.cam.atmos.severe_nbrcnt.last31days.v20230724.tar
evs.plots.cam.atmos.severe_nbrcnt.last90days.v20230724.tar
 evs.plots.cam.atmos.severe_nbrctc.last31days.v20230724.tar
 evs.plots.cam.atmos.severe_nbrctc.last90days.v20230724.tar
evs.plots.cam.atmos.radar_nbrcnt.last31days.v20230729.tar
evs.plots.cam.atmos.radar_nbrcnt.last90days.v20230729.tar
evs.plots.cam.atmos.radar_nbrctc.last31days.v20230729.tar

**Comment by LoganDawson-NOAA** [comment]
@ShelleyMelchior-NOAA I added additional wallclock to the radar job for the nbrctc+last90days job to complete.

You can change lines 18 and 19 of `drive_jevs_cam_radar_plots.sh` to the following to only test that job:
```
LINE_TYPES="nbrctc"
EVAL_PERIODS="LAST90DAYS"
```

**Comment by ShelleyMelchior-NOAA** [comment]
The re-run for just the radar nbrctc last 90 days plots has completed.
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/plots/cam/atmos.20230729`
logfile: `/lfs/h2/emc/ptmp/shelley.melchior/cron.out/jevs_cam_radar_plots_00.o74150132`

**Comment by ShelleyMelchior-NOAA** [comment]
> The re-run for just the radar nbrctc last 90 days plots has completed. COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/plots/cam/atmos.20230729` logfile: `/lfs/h2/emc/ptmp/shelley.melchior/cron.out/jevs_cam_radar_plots_00.o74150132`

@LoganDawson-NOAA Just giving this a bump in case you didn't see this. NOTE: this output is on cactus, from before the prod switch.

**Comment by LoganDawson-NOAA** [comment]
@ShelleyMelchior-NOAA sorry for the delay. the rerun of 90-day radar nbrctc job worked well. The output from all the successful jobs looks good and the logfiles are free of error.

**Comment by ShelleyMelchior-NOAA** [comment]
Excellent! I was doing the code review as we went along. Let me formally finish the code review and approve this PR for merge.

**Review by ShelleyMelchior-NOAA — APPROVED** [review]
Code changes are approved.

**Pull request merged**